src/libYARP_math/src/math.cpp

```diff
@@ -7,34 +7,12 @@
 #include <yarp/os/Log.h>
 #include <yarp/math/Math.h>
 #include <yarp/math/SVD.h>
-#include <gsl/gsl_math.h>
-#include <gsl/gsl_matrix.h>
-#include <gsl/gsl_matrix_double.h>
-
-#include <gsl/gsl_vector.h>
-#include <gsl/gsl_vector_double.h>
-#include <gsl/gsl_blas.h>
 
-#include <gsl/gsl_linalg.h>
-
-#include <gsl/gsl_version.h>
-#include <gsl/gsl_eigen.h>
+#include <cmath>
+#include <gsl/gsl_math.h>
```

Comment:
@lornat75 what is the rationale of this include? `math.cpp` is supposed to be GSL independent now, and any M_PI problem on Windows can be solved independently  from GSL, see https://github.com/robotology/QA/issues/81 .